update quickstart to add k3d

balibebas opened the pull request:
add info on k3s 🙏🏼 

closes #1292

balibebas commented:
Or would it be better to list k3d as a more light-weight docker alternative to KiND?

FxKu commented:
Hm , yeah the description is a bit vague. I have no experience with `k3s`. How can I (easily) test it locally? Would be good if the short description addresses this.

balibebas commented:
> Hm , yeah the description is a bit vague. I have no experience with `k3s`. How can I (easily) test it locally? Would be good if the short description addresses this.

I switched the description to k3d and added more detail. To test locally install k3d, start docker, run `k3d cluster create` and then install the operator.

FxKu commented on docs/quickstart.md:
```diff
@@ -13,6 +13,8 @@ solutions:
   single-node K8s cluster inside a VM (requires KVM or VirtualBox),
 * [kind](https://kind.sigs.k8s.io/), which allows creating multi-nodes K8s
   clusters running on Docker (requires Docker)
+* [k3d](https://k3d.io), which allows creating multi-node K8s clusters running
+  under Docker (requires Docker)
```
```suggestion
* [kind](https://kind.sigs.k8s.io/) or [k3d](https://k3d.io), which allow creating multi-node K8s
  clusters running on Docker (requires Docker)
```

FxKu replied:
lets combine it when it does the same

balibebas replied:
done

balibebas commented:
will squash when ready

FxKu commented:
:+1:

Jan-M commented:
:+1:

balibebas commented:
squashed. ready for merge

FxKu commented:
wasn't necessary. We can squash on merge. Now we have to approve again, but could have told you :smiley:

FxKu commented:
:+1:

sdudoladov commented:
:+1:
